### PR TITLE
Add props for Virtual Items

### PR DESCRIPTION
**Pull request by hsoyier**

### CODE REVIEW

File: src/components/VirtualItem.vue [hunk 1]
```diff
@@ -105,6 +123,14 @@ export default class VitualItem extends Vue {
 
 .s-virtual-item__rarity {
   .margin-top();
+
+  i {
+    margin-right: 8px;
```
Comment (mbiemiller) [inline src/components/VirtualItem.vue]: .margin-right();

File: src/components/VirtualItem.vue [hunk 2]
```diff
@@ -122,7 +148,22 @@ export default class VitualItem extends Vue {
   .flex-centered();
 }
 
+.s-virtual-item__selection-remaining {
+  position: absolute;
+  top: 16px;
+  right: 12px;
+  background: @dark-5;
+  color: @white;
+  .radius(1);
```
Comment (mbiemiller) [inline src/components/VirtualItem.vue]: you dont need 1 - it's the default

File: src/components/VirtualItem.vue [hunk 3]
```diff
@@ -11,13 +11,22 @@
...
+    >
     <span v-if="type" class="s-virtual-item__label">{{ type }}</span>
     <span v-if="quantity" class="s-virtual-item__label">{{ quantity }}</span>
     <div class="s-virtual-item__img">
       <img :src="preview" />
     </div>
     <h3 class="s-virtual-item__name">{{ name }}</h3>
-    <span class="s-virtual-item__rarity">{{ rarity }}</span>
+    <span class="s-virtual-item__rarity" :class="{ entered: isGiveaway }">
+      <i class="icon-check-mark" v-if="this.isGiveaway"></i>
```
Comment (mbiemiller) [inline src/components/VirtualItem.vue]: I dont think you're supposed to use 'this' in markup

**Comment by hsoyier** [comment]
> could this break current integration?
I don't think so, I set v-if for each.

**Comment by hsoyier** [comment]
Fixed all of the mentions about css

**Pull request merged**